### DISCUSSION TITLE
Requests for .model.json

### DISCUSSION THREAD

**Discussion by voghan** [adobe/aem-spa-page-model-manager#50] [Q&A]
When we build our application in production mode we are getting a blank page on our main landing page.  This page doesn't have a path. ie www.oursite.com/.  I see to model.json requests, one for the context root which is correctly loading ourroot.model.json and then a second call that is failing .model.json request.  It seems the process is always looking for the href location to build the mode.json file.  Is there a configuration I am missing here?

**Comment by nitin2io** [comment]
I observed the same issue, we added a change in RouteHelper file to change regex when routePath is '/' it solved the problem of blank page but HTML still not generated when view source only for homepage.
